src/main/java/core/basesyntax/servise/FileReadService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.servise;
+
+import core.basesyntax.ProductsDto;
+import java.util.List;
+
+public interface FileReadService {
+    List<ProductsDto> readFromFile(String fileName);
```

Comment:
it would be even better if you had FileService with methods `read()` and `write()`
and a separate ProductDtoParser service that would parse `List<String>` to `List<ProductDto>`

```suggestion
    List<String> readFromFile(String fileName);
```